### PR TITLE
Added compatibility for RichText chat message bodies; Fixed bad common package import; Fixed all tslint errors;

### PR DESCRIPTION
**Pull request by stef-coenen**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -22,5 +25,8 @@ jobs:
       - name: Install npm dependencies
         run: npm install
 
+      - name: Run linter
+        run: npm run lint
```
Comment (2002Bishwajeet) [inline .github/workflows/test.yml:29]: Should we have another workflow for lints stuff?

**Review by 2002Bishwajeet — CHANGES_REQUESTED** [review]
Is there any changes in the tests (excluding lints) ? Too hard to track down anything

**Comment by stef-coenen** [comment]
> Is there any changes in the tests (excluding lints) ? Too hard to track down anything

Only touched on lints there, didn't touch any test themselves

File: .github/workflows/test.yml (hunk 1 above)
Comment (stef-coenen) [reply .github/workflows/test.yml:29]: Don't think we should, if the lint fails, the test would fail as well. And that seems reasonable right?

Comment (2002Bishwajeet) [reply .github/workflows/test.yml:29]: makes sense

**Review by 2002Bishwajeet — APPROVED** [review]

**Pull request merged**